synapse/storage/data_stores/main/registration.py

```diff
@@ -158,21 +158,44 @@ def set_account_validity_for_user_txn(txn):
         )
 
     @defer.inlineCallbacks
-    def get_expired_users(self):
-        """Get IDs of all expired users
+    def get_expired_users(self, active_only: bool = False):
+        """Get UserIDs of all expired users
+
+        Args:
+            active_only: Whether to return only those users that are marked as active
```

Comment:
The name of this argument sounds really confusing (e.g. "get me all users that are both active and expired" doesn't make a lot of sense), maybe rename it into `active_profiles_only` or something similar?

Reply:
I could also just remove the argument and rename the method considering this method is only used with this option set to `True`.

Reply:
I did the latter.